pkg/netcat/exec.go

```diff
@@ -61,11 +58,9 @@ func ParseCommands(execs ...Exec) (Exec, error) {
 // stdout of the command is send to to the connection
 // stderr of the command is displayed on stdout of the host
 // The host process exits with the exit code of the command unless --keep-open is specified
-func (n *Exec) Execute(stdin io.ReadWriter, stdout io.Writer, stderr io.Writer, eol []byte) error {
-	var (
-		cmd    *exec.Cmd
-		buffer bytes.Buffer
-	)
+func (n *Exec) Execute(stdout io.Writer, stderr io.Writer, eol []byte) error {
+	var cmd *exec.Cmd
+	// buffer bytes.Buffer
```

Comment:
I think we can drop this line completely (last thing I found in this PR, I promise :eyes: )

Reply:
Sure, this was just an oversight on my part.